javascript/ql/src/semmle/javascript/security/dataflow/UnvalidatedDynamicMethodCall.qll

```diff
@@ -17,6 +17,15 @@ module UnvalidatedDynamicMethodCall {
   import UnvalidatedDynamicMethodCallCustomizations::UnvalidatedDynamicMethodCall
   private import DataFlow::FlowLabel
 
+  // Materialize flow labels
```

Comment:
You could add the below as a test somewhere: 

```CodeQL
import javascript

select any(DataFlow::FlowLabel label)
```

That way we have a failing test if new flow-labels are added globally. 

Also, this PR could use a merge. 
And do you intend to rebase this PR on top of the recent `ApiGraph` changes?

Reply:
> And do you intend to rebase this PR on top of the recent ApiGraph changes?

No, mentioned in the PR description, it took a lot of evaluations to get here, so I'll defer integration to a later PR.